openupgradelib/openupgrade.py

```diff
@@ -679,7 +679,26 @@ def rename_fields(env, field_spec, no_deep=False):
                 AND context ~ %(old_pattern)s
             """ % {
                 'old_pattern': (
-                    r"$$('group_by'|'col_group_by'):([^\]]*)"
+                    r"""$$('group_by'|'col_group_by'|'graph_groupbys'|'graph_measure'
+                            |'pivot_measures'|'pivot_row_groupby'|'pivot_column_groupby'
+                        ):([^\]]*)"""
+                    r"'%s(:day|:week|:month|:year){0,1}'(.*?\])$$"
+                ) % old_field,
+                'new_pattern': r"$$\1:\2'%s\3'\4$$" % new_field,
+            }, (model, ),
+        )
+        # TODO: Rename when the field is part of a domain in ir_ui_view_custom
+        cr.execute(r"""
+            UPDATE ir_ui_view_custom
```

Comment:
AFAIK, this can't be done this way, as you don't know for which model the definition is, so a simply renaming by field name can't be done, as you would change an appearance of the same field name in another model.

Reply:
You're right, I will do another PR in the future

Reply:
Have you seen this PR merged yet?